lib/generators/rolemodel/source_map/source_map/app/middleware/rolemodel/source_map_generator.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+module Rolemodel
+  class SourceMapGenerator < Rails::Generators::Base
```

Comment:
Is this supposed to be inherited from a rails generator? This seems like the actual middleware, not the generator.

Reply:
Should this be in a templates folder?